packages/aws-cdk-lib/aws-codepipeline/README.md

```diff
@@ -903,6 +903,10 @@ new codepipeline.Pipeline(this, 'Pipeline', {
 });
 ```
 
+## usePipelineServiceRoleForActions field
```

Comment:
super super nit: Avoid having the property name in the title. Title should be descriptive and the content should mention `usePipelineRoleForActions`.